src/services/swap/providers/lightning-labs/loop-rest-client.ts

```diff
@@ -0,0 +1,134 @@
+// https://lightning.engineering/loopapi/#loop-rest-api-reference
+import https from "https"
+
+import axios, { AxiosResponse } from "axios"
+
+import { getSwapConfig } from "@config"
+
+const loopMacaroon = process.env.LOOP_MACAROON
+  ? convertMacaroonToHexString(Buffer.from(process.env.LOOP_MACAROON, "base64"))
+  : ""
+
+const cert = process.env.LND1_TLS ? Buffer.from(process.env.LND1_TLS, "base64") : ""
+
+const key = process.env.LND1_TLS_KEY
```

Comment:
value fetched from `process.env` should be in a `config` file (where the other `process.env` are)